exercise_perseus_render/assets/src/vue/index.vue

```diff
@@ -1,36 +1,102 @@
 <template>
 
   <div id="exercise-container">
-    <perseus v-if="item" :item="item"></perseus>
+    <assessment-wrapper
+      id = "assessment-container"
+      v-ref:wrapper
+      v-if="exercise"
+      :item-id="itemId"
+      :mastery-model="exercise.mastery_model"
+      :mastery-spacing-time="exercise.masterySpacingTime"
+      :mastery-criterion="exercise.masteryCriterion">
+      <perseus v-if="item" :item="item" :pass-ratio-m="passRatioM" :pass-ratio-n="passRatioN" v-on:nextquestion="nextQuestion" v-on:nextcontent="nextContent"></perseus>
+    </assessment-wrapper>
   </div>
 
 </template>
 
 
 <script>
 
-  const defaultQuestion = require('./question.json');
+  const ss = require('seededshuffle');
```

Comment:
the question.json is a dummy json Richard created for testing, now we just use the real one from Jordan.
seededshuffle is a seeded randomize library. I use the user_id as the seed.